hpecp/client.py

```diff
@@ -142,12 +143,12 @@ def create_from_config_file(
                 "Could not find configuration file '{}'".format(config_file)
             )
 
-        config = configparser.ConfigParser()
-        config.read(config_file)
+        config = SafeConfigParser()
+        config.readfp(codecs.open(config_file, "r", "utf8"))
```

Comment:
@bharath-m-g, I had a lot of issues with 'could not find 'default' section with python 2.7 tests.  This has fixed it for me.